src/routes/home/components/CreateGameDialog.vue

```diff
@@ -29,20 +29,20 @@
         </a>
         {{ t('home.playAiContent2') }}
       </h4>
-      <form name="create_game_form" class="d-flex align-center">
-        <StatsScoringDialog
-          activator-color="surface-2"
-          :show-button-text="false"
-        />
-        <v-switch
-          v-model="isRanked"
-          class="d-flex align-center"
-          :label="isRanked ? t('global.ranked') : t('global.casual')"
-          data-cy="create-game-ranked-switch"
-          color="surface-2"
-        />
-      </form>
       <v-form @submit.prevent="submitNewGame">
```

Comment:
I believe we wanted to wrap the whole #body with the v-form, so from lines 20 - 56

Reply:
I've made the changes you requested, wrapping the entire #body with the <v-form> tag as per the PR description. The commit has been pushed, and everything should now be in line with the requirements.

Please let me know if you need any further adjustments.

Thanks,
Hammad